back/README.md

```diff
@@ -71,13 +71,13 @@ Alternatively, there is a convenience script to reset the whole environment for
 Resetting the database, with the previous command, upsets the testing database as well. Before running the tests, it's sometimes necessary to put it back in it's default shape. We can do this with the following command:
 
 ```
-docker-compose run --rm --user "$(id -u):$(id -g)" -e RAILS_ENV=test web rake db:environment:set db:drop db:create db:schema:load
+docker-compose run --rm --user "$(id -u):$(id -g)" -e RAILS_ENV=test web bundle exec rake db:environment:set db:drop db:create db:schema:load
```

Comment:
Was this just solving a problem, or something extra needed for this new template service?

Reply:
After some recent changes, the command no longer works. We now need to add `bundle exec` each time we want to run a rake task. So this is not related to this PR.